Cargo.toml

```diff
@@ -87,6 +87,8 @@ opt-level = "z"
 lto = true
 
 [patch.crates-io]
+# Patch that enables byte arrays to be returned as strings
+serde_json = { version = "1.0.140", git = "https://github.com/ThetaSinner/json.git", branch = "master" }
```

Comment:
I think it's appropriate to patch `serde_json` here globally... because that's what is required.

Reply:
That is an option for sure. It's a pretty minor change on a fork of the original library. It doesn't release that frequently so it's not a huge problem to have to update the fork. 

It would be nice in a way to just use the fork where it's needed though and let everything else stay on the real version and get upgrades. I wonder if should just use this in the sandbox tool? Maybe even as a renamed crate dependency so that it can literally be used in the zome call code and nowhere else?